Converters/Revit/Speckle.Converters.RevitShared/ToHost/Raw/Geometry/MeshConverterToHost.cs

```diff
@@ -125,14 +123,14 @@ private XYZ[] ArrayToPoints(IList<double> arr, string units)
     }
 
     XYZ[] points = new XYZ[arr.Count / 3];
-    var fTypeId = _scalingServiceToHost.UnitsToNative(units) ?? UnitTypeId.Meters;
+    var fTypeId = scalingServiceToHost.UnitsToNative(units);
```

Comment:
why remove the fallback here, I would not touch it

Reply:
It never returns null so it's redundant.  `UnitsToNative` throws an exception if the value is bad.